wallet-new/integration/MainHardwareWalletSpecs.hs

```diff
@@ -0,0 +1,46 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RecordWildCards   #-}
+module MainHardwareWalletSpecs
+where
+
+import           Universum
+
+import           Cardano.Wallet.API.V1.Types
+--import           Cardano.Wallet.Client.Http as Http
+import           CLI (CLIOptions (..))
+import           HardwareWalletSpecs
+--import           Pos.Core as Core
+import           SetupTestEnv (setupClient)
+--import           Util
+import           Pos.Crypto.Configuration (ProtocolMagic (..),
+                     ProtocolMagicId (..),
+                     RequiresNetworkMagic (RequiresNoMagic))
+
+unSafeProtocolMagic :: ProtocolMagic
+unSafeProtocolMagic = ProtocolMagic
+    { getProtocolMagicId = ProtocolMagicId 55550001
+    , getRequiresNetworkMagic = RequiresNoMagic
+    }
+-- Utility function to run standalone tests outside of Main.
+-- Usefull from a REPL.
+test1 :: Seed -> IO (NewWallet, UnsignedTransaction)
+test1 seed = do
+    (wc,_) <- setupClient options
+    postUnsignedTransactionTest seed wc "srcWallet" "dstWallet"
+
+test2 :: Seed -> IO Transaction
+test2 seed = do
+  (wc,_) <- setupClient options
+  postUnsignedTransactionTest seed wc "srcWallet2" "dstWallet2"
+                        >>= signAndPostTransactionTest unSafeProtocolMagic wc
```

Comment:
`test1` and `test2` :confused: ? May we find better names, especially if they get exported.

Reply:
MainHardwareWalletSpecs only there for convenience
as a quick way to run the tests from the REPL
It it not supposed to be imported.
I don't know if it is OK to have it in the repo but
at latest when new tests are added I would need something similar to this module again.
Technically it is dead code but it may be useful when working on the tests again.

Reply:
I didn't quite catch that this module was not even imported or compiled (not in the .cabal file) ... Let's not commit that then :)